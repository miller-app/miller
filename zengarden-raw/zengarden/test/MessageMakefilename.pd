#N canvas 353 201 637 373 10;
#X obj 450 139 print;
#X obj 450 119 makefilename dog%d.aif;
#X obj 106 21 loadbang;
#X obj 176 21 bng 15 250 50 0 empty empty empty 17 7 0 10 -262144 -1
-1;
#X msg 450 99 0;
#X obj 246 145 print;
#X obj 246 123 makefilename dog%s.aif;
#X obj 43 236 print;
#X obj 42 134 makefilename dog%%d.%s;
#X msg 42 158 set \$1;
#X obj 42 212 makefilename not-set-yet;
#X obj 246 98 symbol meat;
#X obj 42 105 symbol aif;
#X obj 106 54 t b b b b;
#X msg 74 182 0;
#X connect 1 0 0 0;
#X connect 2 0 13 0;
#X connect 3 0 13 0;
#X connect 4 0 1 0;
#X connect 6 0 5 0;
#X connect 8 0 9 0;
#X connect 9 0 10 0;
#X connect 10 0 7 0;
#X connect 11 0 6 0;
#X connect 12 0 8 0;
#X connect 13 0 14 0;
#X connect 13 1 12 0;
#X connect 13 2 11 0;
#X connect 13 3 4 0;
#X connect 14 0 10 0;
