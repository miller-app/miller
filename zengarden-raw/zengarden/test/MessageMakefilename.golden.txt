[@ 0.000ms] print: dog0.aif
[@ 0.000ms] print: dogmeat.aif
[@ 0.000ms] print: dog0.aif